src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -713,6 +713,16 @@ public function myOrderSPaymentMethodShouldBe(PaymentMethodInterface $paymentMet
     }
 
     /**
+     * @When /^I complete addressing step with email "([^"]*)" and (address "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)" for "([^"]+)")$/
```

Comment:
Ayyy carramba, address matching looks too complicated, what about `/^I complete addressing step with email "([^"]*)" and (address .+?)$/`?